lib/jnpr/junos/utils/sw.py

```diff
@@ -967,11 +967,14 @@ def reboot(self, in_min=0, at=None, all_re=True):
 
         .. todo:: need to better handle the exception event.
         """
+        if hasattr(self._dev, '_is_linux') and self.dev.facts['_is_linux']:
```

Comment:
I don't understand this line. There is no `self._dev._is_linux` attribute, and there is no `self.dev` attribute.
I believe this line should be: `if self._dev.facts['_is_linux']:`